reverbePrintemps/spiced
# Fix bugs with survey

# Context

This PR aims to [fix the bugs within the survey page](https://github.com/spiced-academy/honey-web-dev/blob/main/sessions/js-basics/challenges-js-basics.md#fix-the-errors-in-a-survey-app).

# Changes

- [x] Fixed all bugs described in [Bugs](#bugs)

# Bugs

1. "This code selects the "next" button using an ID. This is bad practice. Can you find a better solution?"
2. "When clicking the "previous" button, this event listener is not executed. Do you know what's wrong with this code?"
3. "This event listener is connected properly. When clicked the button is shown active. However, the debug message doesn't appear in the debug console. Where is the error with this code?"
4. "When clicking on the second choice, the button should be shown with an active state. Do you see why it isn't working?"

# How to run

1. Clone repo
2. Checkout this branch
3. Run in the browser

# Test / expected results

- Click on each survey answer => notice green active state + corresponding message in console
- Click on next/previous buttons => notice message in the console

# TODO


# GIF

![](https://media.giphy.com/media/11ZSwQNWba4YF2/giphy.gif)
